packages/fluentui/react-northstar/src/components/Menu/MenuItem.tsx

```diff
@@ -320,7 +320,7 @@ export const MenuItem = compose<'a', MenuItemProps, MenuItemStylesProps, {}, {}>
           trySetMenuOpen(false, e, () => focusAsync(itemRef.current));
         } else {
           // the menuItem element was clicked => toggle the open/close and stop propagation
-          trySetMenuOpen(active ? !menuOpen : true, e);
+          trySetMenuOpen(active && on !== 'hover' ? !menuOpen : true, e);
```

Comment:
Could you add a test for that, please? :)

Reply:
Done